runtime/util/ap32/volatile.s

```diff
@@ -131,7 +131,7 @@ TOC.longVolatileWrite: .tc .longVolatileWrite[tc],longVolatileWrite[ds]
 # Optimized case: Running 32 bit vm on 64 bit hw can use FP registers
         lfd fp3,0(r4)
 # Find and align volatileSwapArea
-        addi r3,r1,(7+88)                        #   optimized to ternary
+        addi r3,r1,(7+88)                        #   optimized to select
```

Comment:
I don't believe these comments are referring to the "select" IL opcodes.  Please change it back.  Applies to all changes in this file.